packages/termux-tools/termux-change-repo

```diff
@@ -196,6 +196,11 @@ replace_repository() {
     fi
 }
 
+if command -v apt 1>/dev/null; then
+    echo "Error: changing mirrors can't execute because apt is not installed."
```

Comment:
I would suggest to use following message:
```
Error: Changing mirror requires apt. Install it using 'pkg i apt -y'
```
It seems more comprehensive and simple to me.

*This is just an suggestion, please do not take it otherwise.*

Reply:
Also, I would suggest to write commit message in imperative sentence. You may use something like:
```
enhance(main/termux-tools:termux-change-repo): fail if apt isn't installed
```